routes/api.php

```diff
@@ -1,3 +1,24 @@
 <?php
 
 declare(strict_types=1);
+
+use App\Http\Controllers\DonateController;
+use App\Http\Controllers\iDealController;
+use Illuminate\Support\Facades\Route;
+
+// JSON API's for website
+Route::controller('ApiController')->group(function () {
```

Comment:
Controller groups zijn leuk in theorie, maar in de praktijk vrij buggy. De string referentie hier werkt, maar vervangen door `ApiController::class` niet (dan gaat de namespace compleet naar de tering en kan ie niet resolved worden).

Ook getest met de routes voor `iDealController` maar dan werken de `action()` url helpers niet meer. Nou ja, dan maar zo laten.

Reply:
Dit komt omdat in de routing service er een namespace geprefixed wordt. Dat is OUD! (zie laatste commit ;) )

Reply:
Goed gevonden!